src/site/_filters/md.js

```diff
@@ -16,7 +16,13 @@ module.exports = (content, opts) => {
     md.set(opts);
   }
 
-  return content.split('\n').length > 1
-    ? md.render(content)
-    : md.renderInline(content);
+  let inline = !content.includes('\n');
+
+  // If there's quite a bit of content, we want to make sure
```

Comment:
I need to double check if this breaks anything on the site. Our screenshot coverage with Percy isn't very comprehensive.

Reply:
reading through the code again, this _seems_ ok... let's go with it and tinker if something looks off.

Reply:
Yeh I reckon it'll be fine. It's just catching cases where you get like a paragraph of text that accidentally gets no formatting